api/src/api/post/post.routes.ts

```diff
@@ -0,0 +1,13 @@
+import express from "express";
+
+import { CreatePost, GetPosts, GetPhotos, UpdatePost, DeletePost } from "./post.controllers";
+
+const router = express.Router();
+
+router.post("/createpost", CreatePost);
+router.get("/getposts", GetPosts);
+router.post("/getphotos", GetPhotos);
+router.post("/updatepost", UpdatePost);
+router.post("/deletepost", DeletePost);
```

Comment:
Same thing above:
- `GET posts/:id`: Get a post
- `PUT posts/:id`: Update a post
- `DELETE posts/:id`: Delete a post 
- `POST posts`: Create a post
- `GET posts`: Get ALL posts (for the home page). Normally we would have pagination but we can implement that later.

There is no need for having `getphotos` or `getcomments`.  Posts returned by `GET post/:id`  and `GET posts` should have everything under it in the JSON response. See [Prisma relation queries guide](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries) on how to do that.